gettsim/dag.py

```diff
@@ -182,18 +182,19 @@ def fail_if_functions_and_columns_overlap(func_dict, data):
 
                     {formatted}
 
-                {'This is' if n_cols == 1 else 'These are'} already present among the functions of the taxes and
-                transfers system.
+                {'This is' if n_cols == 1 else 'These are'} already present among the
+                functions of the taxes and transfers system.
 
-                If you want {'this' if n_cols == 1 else 'a'} data column to be used instead of calculating it within
-                GETTSIM, please specify it among the *user_columns*.
+                If you want {'this' if n_cols == 1 else 'a'} data column to be used
```

Comment:
This is precisely what I wanted to avoid, sorry. The message the user is getting will look ugly because this line and others are too short. 

Maybe define extra variables upfront?